google/cloud/firestore_v1/collection.py

```diff
@@ -430,7 +431,7 @@ def on_snapshot(self, callback):
             collection_ref = db.collection(u'users')
 
             def on_snapshot(collection_snapshot, changes, read_time):
-                for doc in collection_snapshot.documents:
+                for doc in collection_snapshot:
```

Comment:
In here you're deleting `documents`, but in the next file you're keeping it, though examples looks the same

Reply:
This given example is not working , if i remove `.documents` then it works properly as it gives document_snapshot in return, so once i confirm it,i will changed it.